SU2_CFD/src/variables/CAdjEulerVariable.cpp

```diff
@@ -37,9 +37,8 @@
 
 #include "../../include/variables/CAdjEulerVariable.hpp"
 
-
 CAdjEulerVariable::CAdjEulerVariable(su2double psirho, const su2double *phi, su2double psie, unsigned long npoint, unsigned long ndim,
-                                     unsigned long nvar, CConfig *config) : CVariable(npoint, ndim, nvar, config) {
+                                     unsigned long nvar, CConfig *config) : CVariable(npoint, ndim, nvar, config), Gradient_Reconstruction(Gradient) {
```

Comment:
In c++11 you can have the initializer in the hpp file `VectorOfMatrix& Gradient_Reconstruction = Gradient;`

Reply:
Done